core-plugins/docs/AzureBlobStore-batchsource.md

```diff
@@ -17,7 +17,7 @@ Properties
 **referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
 
 **path:** The path on Microsoft Azure Blob Storage to use as input. The path uses filename expansion (globbing) to read
-files. (Macro-enabled)
+files. The path must strat with `wasb://` or `wasbs://`. (Macro-enabled)
 
 **account:** The Microsoft Azure Blob Storage account to use. This is of the form 
 `mystorageaccount.blob.core.windows.net`, where `mystorageaccount` is the Microsoft 
```

Comment:
It might be better to change account to `mystorageaccount` only and later append '.blob.core.windows.net'.  It can be more consistent with Azure site and the usage of other plug-ins (when I put this plug-in source with other actions together,  it looks better to use on macro across the pipeline).